packages/tools/generator-component/generators/app/templates/__package__.json

```diff
@@ -33,9 +33,9 @@
     "lint:fix": "yarn lint --fix"<% if(config.isComponent) { %>,
     "lint:style": "vue-cli-service lint:style"<% } %>,
```

Comment:
is there anything else we need to change in the generator? Is it worth adding the `@browserstack` tag?

Reply:
I did think this, but would require changing more files. Happy to do this in another PR once we've got a consensus on what tests should be run against BS